x-pack/plugins/observability/public/components/shared/exploratory_view/hooks/use_add_to_case.test.tsx

```diff
@@ -44,19 +43,11 @@ describe('useAddToCase', function () {
     const { setData, core, findByText } = setupTestComponent();
 
     expect(setData).toHaveBeenLastCalledWith({
-      createCaseUrl: '/app/observability/cases/create',
-      goToCreateCase: expect.any(Function),
       isCasesOpen: false,
       isSaving: false,
       onCaseClicked: expect.any(Function),
       setIsCasesOpen: expect.any(Function),
     });
-    fireEvent.click(await findByText('Add new case'));
-
-    expect(core.application?.navigateToApp).toHaveBeenCalledTimes(1);
-    expect(core.application?.navigateToApp).toHaveBeenCalledWith('observability', {
-      path: '/cases/create',
-    });
```

Comment:
I don't understand why we are removing these tests?

Reply:
Yes, you are right, this test should've been adapted, not removed. I am fixing it 👍